### PR TITLE
Update connectors-2.1 features

### PR DESCRIPTION
**Pull request by KyleAure**
Update the connectors-2.1 features to use the connectors-2.1 api instead of the connectors-2.0 api.

### CODE REVIEW

**Comment by wraschke** [comment]
#libby

**Comment by KyleAure** [comment]
#build

File: dev/com.ibm.ws.jca_fat/fat/src/com/ibm/ws/jca/fat/app/DependantApplicationTest.java [hunk 1]
```diff
@@ -90,7 +92,7 @@ private void runTest(String testName) throws Exception {
 
     @BeforeClass
     public static void setUp() throws Exception {
-        server = FATSuite.getServer();
+//        server = FATSuite.getServer();
```
Comment (tabishop) [inline dev/com.ibm.ws.jca_fat/fat/src/com/ibm/ws/jca/fat/app/DependantApplicationTest.java]: nit: this comment can be deleted

File: dev/com.ibm.ws.jca_fat_configprops/fat/src/com/ibm/ws/jca/fat/configprops/FATSuite.java [hunk 2]
```diff
@@ -22,10 +22,14 @@
 @RunWith(Suite.class)
 @SuiteClasses(JCAConfigPropsTest.class)
 public class FATSuite {
+
```
Comment (tabishop) [inline dev/com.ibm.ws.jca_fat_configprops/fat/src/com/ibm/ws/jca/fat/configprops/FATSuite.java:25]: nit: can update copyright year to 2022

File: dev/com.ibm.ws.jca_fat/fat/src/com/ibm/ws/jca/fat/regr/JCAFATTest.java [hunk 3]
```diff
@@ -63,21 +63,18 @@ public abstract class JCAFATTest {
     public final TestName testName = new TestName() {
         @Override
         public void failed(Throwable e, Description description) {
-            System.out.println(timestamp() + " <--- "
-                               + description.getMethodName() + " failed:");
+            System.out.println(timestamp() + " <--- " + description.getMethodName() + " failed:");
```
Comment (tabishop) [inline dev/com.ibm.ws.jca_fat/fat/src/com/ibm/ws/jca/fat/regr/JCAFATTest.java]: nit: can update copyright year

File: dev/com.ibm.ws.jca_fat/fat/src/com/ibm/ws/jca/fat/app/DependantApplicationTest.java [hunk 4]
```diff
@@ -41,10 +41,11 @@
 import com.ibm.websphere.simplicity.config.context.ClassloaderContext;
 import com.ibm.websphere.simplicity.config.context.JEEMetadataContext;
 import com.ibm.websphere.simplicity.log.Log;
-import com.ibm.ws.jca.fat.FATSuite;
 
 import componenttest.annotation.AllowedFFDC;
+import componenttest.annotation.Server;
```
Comment (tabishop) [inline dev/com.ibm.ws.jca_fat/fat/src/com/ibm/ws/jca/fat/app/DependantApplicationTest.java]: nit: can update copyright year

File: dev/com.ibm.ws.jca_fat/fat/src/com/ibm/ws/jca/fat/app/ConnectionManagerMBeanTest.java [hunk 5]
```diff
@@ -36,16 +36,18 @@
 import com.ibm.websphere.simplicity.config.JavaPermission;
 import com.ibm.websphere.simplicity.config.ServerConfiguration;
 import com.ibm.websphere.simplicity.log.Log;
-import com.ibm.ws.jca.fat.FATSuite;
 
+import componenttest.annotation.Server;
```
Comment (tabishop) [inline dev/com.ibm.ws.jca_fat/fat/src/com/ibm/ws/jca/fat/app/ConnectionManagerMBeanTest.java]: nit: can update copyright year

**Review by tabishop — APPROVED** [review]
Just a few nits that I wouldn't hold up approval for

**Comment by KyleAure** [comment]
#build

**Comment by KyleAure** [comment]
#build

**Comment by KyleAure** [comment]
#build

**Comment by KyleAure** [comment]
#build

**Review by tabishop — APPROVED** [review]

**Comment by KyleAure** [comment]
#libbybot

File: dev/com.ibm.ws.jca_fat/fat/src/com/ibm/ws/jca/fat/FATSuite.java [hunk 6]
```diff
@@ -41,14 +42,19 @@ public class FATSuite {
...
+     * EE9 will run in full mode only.
+     * EE10 will run in full and lite mode.
      */
     @ClassRule
     public static RepeatTests r = RepeatTests.with(new EmptyAction().fullFATOnly())
-                    .andWith(new JakartaEE9Action().withWiden()); // need widen option to handle jar file within a jar file.
+                    // need widen option to handle jar file within a jar file.
+                    .andWith(new JakartaEE9Action().withWiden())
+                    // need widen option to handle jar file within a jar file.
+                    .andWith(new JakartaEE10Action().withWiden());
```
Comment (tevans78) [inline dev/com.ibm.ws.jca_fat/fat/src/com/ibm/ws/jca/fat/FATSuite.java]: I think EE9 will run in Lite at the moment ... should it have `fullFATOnly()` on it?

File: dev/io.openliberty.jakarta.connectors.2.1/.classpath [hunk 7]
```diff
@@ -0,0 +1,6 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<classpath>
+	<classpathentry kind="con" path="aQute.bnd.classpath.container"/>
+	<classpathentry kind="con" path="org.eclipse.jdt.launching.JRE_CONTAINER/org.eclipse.jdt.internal.debug.ui.launcher.StandardVMType/JavaSE-1.8"/>
```
Comment (tevans78) [inline dev/io.openliberty.jakarta.connectors.2.1/.classpath]: Shouldn't this be Java 11?

File: dev/io.openliberty.jakarta.connectors.2.1/bnd.bnd [hunk 8]
```diff
@@ -0,0 +1,32 @@
...
+# which accompanies this distribution, and is available at
+# http://www.eclipse.org/legal/epl-v10.html
+#
+# Contributors:
+#     IBM Corporation - initial API and implementation
+#*******************************************************************************
+-include= ~../cnf/resources/bnd/bundle.props
+bVersion=1.0
+
+Bundle-SymbolicName: io.openliberty.jakarta.connectors.2.1; singleton:=true
```
Comment (tevans78) [inline dev/io.openliberty.jakarta.connectors.2.1/bnd.bnd]: Is this bnd file missing the java 11 bits?
```
javac.source: 11
javac.target: 11

Require-Capability: osgi.ee; filter:="(&(osgi.ee=JavaSE)(version=11))"
```

Comment (tevans78) [reply dev/io.openliberty.jakarta.connectors.2.1/bnd.bnd]: Also, I don't think you need this project at all. This API bundle can be folded into `io.openliberty.jakarta`

Comment (tevans78) [reply dev/io.openliberty.jakarta.connectors.2.1/bnd.bnd]: it is already there - https://github.com/OpenLiberty/open-liberty/blob/integration/dev/io.openliberty.jakarta/connector.2.1.bnd

**Comment by KyleAure** [comment]
#build

**Comment by KyleAure** [comment]
Personal Build: failed with an existing defect, and a child build that was never completed.  Restarted child build.

**Comment by KyleAure** [comment]
#libbybot

**Comment by KyleAure** [comment]
Child build passed: https://wasrtc.hursley.ibm.com:9443/jazz/web/projects/WS-CD#action=com.ibm.team.build.viewResult&id=_TVnQYAzhEe2vacLbpVXjpQ

**Pull request merged**